source/modules/MEI.analytical.xml

```diff
@@ -242,17 +242,28 @@
   </classSpec>
   <classSpec ident="att.keySig.anl" module="MEI.analytical" type="atts">
     <desc xml:lang="en">Analytical domain attributes.</desc>
-    <classes>
-      <memberOf key="att.accidental"/>
-      <memberOf key="att.pitch"/>
-    </classes>
```

Comment:
I don’t think we should duplicate the definition of `pname` locally here. There doesn’t seem to be any difference between the attribute class that is being removed and the attribute definition that is being added.